### DISCUSSION TITLE
能不能提供的发信/收信功能文档

### DISCUSSION THREAD

**Discussion by Muska-Ami** [DreamVoid/MiraiMC#189] [问答 / Q&A] [answered]
旧的MiraiMC API可以看之前的模板，新版JavaDoc里一堆类也不知道咋用，能不能写个文档专门讲发信/收信

或者开个储存库当社区非官方文档之类的

这样也便于开发者开发

**Comment by DreamVoid** [comment, marked as answer]
新的Javadoc只需要盯死你的平台就行了，其他用不到的就当不存在，机器人操作还是MiraiBot这一个。事件那些我是跟着官方的顺序来的，想要什么事件直接Ctrl+F搜索就行

非官方文档就不是我来弄了

**Reply by Muska-Ami** [reply]
对了MiraiMC的模板是不是该更新pom.xml了
还有其他的比如Chat2QQ

**Reply by DreamVoid** [reply]
> 对了MiraiMC的模板是不是该更新pom.xml了 还有其他的比如Chat2QQ

没发正式版暂时不更新
